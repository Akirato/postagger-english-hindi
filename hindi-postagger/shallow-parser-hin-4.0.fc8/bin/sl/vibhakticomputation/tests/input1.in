1	((	NP	<fs af='rAma,n,,,,,,' head="rAma">
1.1	rAma	NN	<fs af='rAma,n,,,,,,' name="rAma'>
1.2	ke	PSP	<fs af='ke,psp,,,,,,'>
1.3	hI	RP	<fs af='hI,avy,,,,,,'>
1.4	dvArA	PSP	<fs af='dvArA,psp,,,,,,'>
	))
1	((	NP	<fs af='rAma,n,,,,,,' head="rAma">
1.1	rAma	NN	<fs af='rAma,n,,,,,,' name="rAma'>
1.2	jI	RP	<fs af='jI,n,,,,,,'>
1.2	ke	PSP	<fs af='ke,psp,,,,,,'>
1.3	hI	RP	<fs af='hI,avy,,,,,,'>
1.4	dvArA	PSP	<fs af='dvArA,psp,,,,,,'>
	))
