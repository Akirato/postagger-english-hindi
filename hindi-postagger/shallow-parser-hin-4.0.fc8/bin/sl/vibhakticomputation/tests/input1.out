1	((	NP	<fs af='rAma,n,,,,,0_ke_dvArA,' head="rAma" vpos="2_RP_4">
1.1	rAma	NN	<fs af='rAma,n,,,,,,' name="rAma'>
1.2	hI	RP	<fs af='hI,avy,,,,,,'>
	))		
2	((	NP	<fs af='rAma,n,,,,,0_ke_dvArA,' head="rAma" vpos="3_RP_5">
2.1	rAma	NN	<fs af='rAma,n,,,,,,' name="rAma'>
2.2	jI	RP	<fs af='jI,n,,,,,,'>
2.3	hI	RP	<fs af='hI,avy,,,,,,'>
	))		
